docs/azure_arc_jumpstart/azure_arc_sqlsrv/azure/azure_arm_template_winsrv/_index.md

```diff
@@ -39,13 +39,15 @@ By the end of the guide, you will have an Azure VM installed with Windows Server
 
     ```shell
     az login
-    az ad sp create-for-rbac -n "<Unique SP Name>" --role contributor
+    az account set --subscription "<Subscription ID>"
+    az ad sp create-for-rbac -n "<Unique SP Name>" --role contributor --scopes /subscriptions/<Subscription ID>
     ```
 
     For example:
 
     ```shell
-    az ad sp create-for-rbac -n "http://AzureArcServers" --role contributor
+    az account set --subscription "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxx"
+    az ad sp create-for-rbac -n "http://AzureArcServers" --role contributor --scopes /subscriptions "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxx"
```

Comment:
For this section, can we use the same language found in ArcBox readmes? For example, the SP name should not have http:// in front. There is also script code to automatically pull the subscription id.

Reply:
@dkirby-ms and I worked through this together on a call and have applied the necessary code changes.